### PR TITLE
Move buffer components

### PR DESCRIPTION
**Pull request by hharnisc**
Bring Buffer Components repo into Buffer Publish :raised_hands:

### CODE REVIEW

File: packages/components/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Purpose
```
Comment (alvarezmelissa87) [inline packages/components/.github/PULL_REQUEST_TEMPLATE.md]: This seems like a weird thing to be added 🤔

Comment (hharnisc) [inline packages/components/.github/PULL_REQUEST_TEMPLATE.md]: Right on - deleting that

**Comment by hharnisc** [comment]
Likewise on the review @alvarezmelissa87 :rocket: 

Pending :green_apple: build I'll merge this one in

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +0,0 @@
-[submodule "packages/components"]
```
Comment (emily-plummer) [inline .gitmodules]: 🎉

**Pull request merged**